lighthouserc.cjs

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    ci: {
+      upload: {
+        target: 'temporary-public-storage',
+      },
+      collect: {
+        staticDistDir: './dist',
+        "url": [
+          "http://localhost/"
+        ]
+      },
+      asserts: {
+        assertions: {
+          'categories:accessibility': ['warn', { minScore: 1 }],
+          'categories:best-practices': ['warn', { minScore: 1.0 }],
+          'categories:performance': ['warn', { minScore: 0.2 }],
```

Comment:
Was our performance that bad 🤣

Reply:
I think 0.9 is a good benchmark for SEO and accessibility, while 0.8 is better for best-practices and SEO. What do you think?

Reply:
Sounds good!